SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -1712,6 +1771,55 @@ USpatialActorChannel* ActorSystem::TryRestoreActorChannelForStablyNamedActor(AAc
 	return Channel;
 }
 
+bool ActorSystem::InvokePreNetReceive(USpatialActorChannel& Channel, UObject& Object)
+{
+	if (Object.IsPendingKill())
+	{
+		UE_LOG(LogActorSystem, Log,
+			   TEXT("InvokePreNetReceive: Did not invoke PreNetReceive for object %s, as object is pending kill. Entity id: %lld."),
+			   *Object.GetName(), Channel.GetEntityId());
+		return false;
+	}
+
+	if (!PostNetReceivesToSend.Contains(FWeakObjectPtr(&Object)))
```

Comment:
Maybe a comment explaining why this is what we want and why we need it.
E.g. need this to only call prenetreceives once, blabla